src/components/navigation/header/config/DrawerNodeItems.tsx

```diff
@@ -7,9 +7,26 @@ import { CartIcon } from 'src/components/icons';
 
 import { NavigationLinkProps } from '../links/NavigationLink';
 import { shouldShowComparisonLink } from '../ComparisonItem';
+import {
+  assure,
```

Comment:
Not sure `assure` is the right word in the context of  insurance

Reply:
translation key and item itself is also called like that 😓

Reply:
Alright lets keep it consistent then (unless you have the energy to rename everywhenre)